_studio/mfx_lib/encode_hw/h265/src/mfx_h265_encode_vaapi.cpp

```diff
@@ -1302,9 +1304,9 @@ bool FillCUQPDataVA(Task const & task, MfxVideoParam &par, CUQPMap& cuqpMap)
             return  false;
         }
         // Fill complete buffer, because of LCU based averaging
-        for (mfxU32 i = 0; i < cuqpMap.m_h_aligned; i++)
+        for (mfxU32 i = 0; i < inputH; i++)
         {
-            for (mfxU32 j = 0; j < cuqpMap.m_pitch; j++)
+            for (mfxU32 j = 0; j < inputW; j++)
             {
```

Comment:
Accessed buffer cuqpMap.m_buffer is cuqpMap.m_h_aligned by cuqpMap.m_pitch. Need to check if LCU aligned input* values are <=. Provided dimension can be changed.